app/models/operatingsystem.rb

```diff
@@ -72,13 +72,17 @@ class Operatingsystem < ApplicationRecord
                'Xenserver' => %r{XenServer}i }
 
   class Jail < Safemode::Jail
-    allow :name, :media_url, :major, :minor, :family, :to_s, :repos, :==, :release, :release_name, :kernel, :initrd, :pxe_type, :medium_uri, :boot_files_uri, :password_hash
+    allow :name, :media_url, :major, :minor, :family, :to_s, :==, :release, :release_name, :kernel, :initrd, :pxe_type, :medium_uri, :boot_files_uri, :password_hash, :additional_media
```

Comment:
I am getting errors in safe mode. Symbol `:additional_media` needs to be moved from here to `DEFAULT_ALLOWED_VARIABLES` in `Renderer::Configuration`. These are allowed *methods* for an operating system.

Not sure why you deleted `repos` from there, is this intentional?

Reply:
`repos` was previously unused and I removed that in this PR. It could affect a plugin if it provides `@repos`, but none of our templates ever even looked at that.

additional_media replaces the functionality.

Reply:
Sure, but why you keep `additional_media` in the allow list for OS here? Is your intention to allow users to call it with provider directly?

Reply:
No, I suppose there's little value in having it here, I was mostly just keeping the same semantics as `repos`.  I've removed it.